tests/testthat/test-individual.R

```diff
@@ -136,3 +136,25 @@ test_that("resource_compensation_point", {
     expect_equal(p$resource_compensation_point(), resource_compensation_point_R(x, p), tolerance=1e-5)
   }
 })
+
+
+test_that("Maximise individual outcome", {
+  #set bounds
+  bounds = bounds(lma=c(0.01, 0.08))
+  
+  size_res <- optimise_individual_rate_at_size_by_trait(bounds, log_scale = TRUE, tol = 0.001, size = 3, type = names(strategy_types)[x], size_name = "height")
+  #check outcome real and positive
+  expect_true(res > 0)
```

Comment:
Add a test which looks at the optimised value and tests against this

e.g. 

`expect_equal(height_res$value, 0.56)`

where `height_res` is the output traits, and 0.56 is the values of the trait you get running this and are satisfied it's reasonable. 

The purpose of hardcoding a test like this is to ensure that down the track the model keeps producing the same result.

Reply:
Have implemented a hardcode for both height growth rate and area heartwood growth rate. Unfortunately falls down when iterating across FF16 and FF16r, but only for area heartwood growth rate, but surprisingly not height growth rate. Could make hardcoded tests unique to certain stratgies?

Reply:
> Could make hardcoded tests unique to certain strategies?
Good idea. Just add an if statement

Reply:
This error test seems to be failing on Windows and Ubuntu, but not Mac, is that unusual?

Reply:
Nice work!

Looking at the failure, the results aren't that different.

This highlights the difference in numerical results across operating systems. 

https://github.com/traitecoevo/plant/actions/runs/7648848770/job/20842305908#step:6:1

So just increase the tolerance allowed in the test.